neuralop/training/trainer.py

```diff
@@ -149,27 +172,35 @@ def train(self, train_loader, test_loaders,
                     sample = self.data_processor.preprocess(sample)
                 else:
                     # load data to device if no preprocessor exists
-                    sample = {k:v.to(self.device) for k,v in sample.items() if torch.is_tensor(v)}
+                    sample = {
+                        k: v.to(self.device)
+                        for k, v in sample.items()
+                        if torch.is_tensor(v)
+                    }
 
                 if self.amp_autocast:
                     with amp.autocast(enabled=True):
-                        out  = self.model(**sample)
+                        out = self.model(**sample)
                 else:
-                    out  = self.model(**sample)
+                    out = self.model(**sample)
 
                 if self.data_processor is not None:
                     out, sample = self.data_processor.postprocess(out, sample)
 
                 if self.callbacks:
                     self.callbacks.on_before_loss(out=out)
 
-                loss = 0.
+                loss = 0.0
 
                 if self.overrides_loss:
                     if isinstance(out, torch.Tensor):
-                        loss += self.callbacks.compute_training_loss(out=out.float(), **sample, amp_autocast=self.amp_autocast)
+                        loss += self.callbacks.compute_training_loss(
+                            out=out.float(), **sample, amp_autocast=self.amp_autocast
+                        )
                     elif isinstance(out, dict):
-                        loss += self.callbacks.compute_training_loss(**out, **sample, amp_autocast=self.amp_autocast)
+                        loss += self.callbacks.compute_training_loss(
+                            **out, **sample, amp_autocast=self.amp_autocast
```

Comment:
I wonder if it is not confusing to have out in one case and **out, **sample in the other

Reply:
changed this, as we never implemented a dict